log-tools/elastic/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+services:
+  es01:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
```

Comment:
is not better to use https://opensearch.org/ ?

Reply:
we are not using any paid things and we are not hosting it. do you have any concerns with it?

Reply:
currently not a lot, but considering history of elasticsearch is hard to know what they do in future, so in general i think we should use rather opensearch, but we can change it later